toolkit/scripts/mariner-required-configs.json

```diff
@@ -0,0 +1,640 @@
+{
+    "required-configs": {
+        "CONFIG_INTEL_IOMMU_SVM": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
+        },
+        "CONFIG_DRM_AMDGPU_SI": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5416"
+        },
+        "CONFIG_TLS": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5406"
+        },
+        "CONFIG_NVME_TARGET_TCP": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5406"
+        },
+        "CONFIG_DELL_WMI_DESCRIPTOR": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5406"
+        },
+        "CONFIG_X86_PLATFORM_DRIVERS_DELL": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/2671"
+        },
+        "CONFIG_DRM_AMDGPU": {
+            "value": [
+                "m",
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": "https://github.com/microsoft/CBL-Mariner/pull/5416"
+        },
+        "CONFIG_BINFMT_MISC": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
+        },
+        "CONFIG_SCHEDSTATS": {
+            "value": [
+                "y"
+            ],
+            "arch": [
+                "AMD64"
+            ],
+            "comment": "Needed for customer",
+            "PR": null
```

Comment:
>"PR": null [](http://example.com/codeflow?start=12&length=10)

Suggestion: for changes, that were done before Mariner GA'ed and got moved to GitHub, we could either mention the PR number (1693 in this case) from those times or just say "Initial Mariner kernel configuration".